concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -286,6 +288,7 @@ def test_that_concent_responds_with_service_refused_when_subtask_results_rejecte
         )
         self._assert_stored_message_counter_not_increased()
 
+    @override_settings(CELERY_TASK_ALWAYS_EAGER=True)
```

Comment:
Instead of this please create mock for `blender_verification_request.delay` and add assert that it was called with exact values (use `assert_called_with`). This is actually quite important to have this call tested.

Reply:
Done.